[2025-03-21T19:07:02.940+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-21T19:07:02.965+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather-data-pipeline.create_insert_mart_tables scheduled__2025-03-21T18:00:00+00:00 [queued]>
[2025-03-21T19:07:02.981+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather-data-pipeline.create_insert_mart_tables scheduled__2025-03-21T18:00:00+00:00 [queued]>
[2025-03-21T19:07:02.982+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 2
[2025-03-21T19:07:02.997+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): create_insert_mart_tables> on 2025-03-21 18:00:00+00:00
[2025-03-21T19:07:03.005+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1548) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-21T19:07:03.007+0000] {standard_task_runner.py:72} INFO - Started process 1550 to run task
[2025-03-21T19:07:03.008+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather-data-pipeline', 'create_insert_mart_tables', 'scheduled__2025-03-21T18:00:00+00:00', '--job-id', '155', '--raw', '--subdir', 'DAGS_FOLDER/weather-data-pipeline.py', '--cfg-path', '/tmp/tmpr91bjnxg']
[2025-03-21T19:07:03.011+0000] {standard_task_runner.py:105} INFO - Job 155: Subtask create_insert_mart_tables
[2025-03-21T19:07:03.348+0000] {task_command.py:467} INFO - Running <TaskInstance: weather-data-pipeline.create_insert_mart_tables scheduled__2025-03-21T18:00:00+00:00 [running]> on host c25e9da6226b
[2025-03-21T19:07:03.487+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather-data-pipeline' AIRFLOW_CTX_TASK_ID='create_insert_mart_tables' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T18:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-21T18:00:00+00:00'
[2025-03-21T19:07:03.488+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-21T19:07:03.512+0000] {base.py:84} INFO - Retrieving connection 'rizqi-neon'
[2025-03-21T19:07:04.365+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-21T19:07:04.383+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-21T19:07:04.384+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weather-data-pipeline, task_id=create_insert_mart_tables, run_id=scheduled__2025-03-21T18:00:00+00:00, execution_date=20250321T180000, start_date=20250321T190702, end_date=20250321T190704
[2025-03-21T19:07:04.512+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-21T19:07:04.713+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-21T19:07:04.726+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
